<?xml version="1.0"?>
<launch>
    <node pkg="twist_controller" type="dbw_node.py" name="dbw_node">
        <param name="control_update_frequency" value="50" />
        <param name="vehicle_mass" value="1736.35" />
        <param name="fuel_capacity" value="13.5" />
        <param name="brake_deadband" value=".1" />
        <param name="decel_limit" value="-8.5" /> <!-- 8.5m/s^2 is the general limit-->
        <param name="accel_limit" value="2.5" /> <!-- 2.5m/s^2 for general vehicle, racing vehicle can reach 3.8m/s^2 or above -->
        <param name="wheel_radius" value="0.2413" />
        <param name="wheel_base" value="2.8498" />
        <param name="steer_ratio" value="14.8" />
        <param name="max_lat_accel" value="3." />
        <param name="max_throttle" value="0.8" />
        <param name="max_steer_angle" value="8." />
    </node>
</launch>
